src/js/idx-omnibar.js

```diff
@@ -45,15 +45,29 @@ var idxOmnibar = function(jsonData){
 		}
 		return displayName;
 	};
+
+	let includedStateAbreviations = new Set();
+
 	//helper function for grabbing the name of each item in JSON creating new array
 	var createArrays = function(array, newArray, type, fieldName){
+		
+		// Add state abbreviations to the set for later reference
+		array.forEach((item) => {
+			if (item.stateAbrv) {
+				console.log(item.stateAbrv);
```

Comment:
I don't think this console log is needed.  Everything else looks like it works fine! Good stuff